conda.recipe/meta.yaml

```diff
@@ -15,31 +15,40 @@ source:
 requirements:
   build:
```

Comment:
When only enabling `python` and `setuptools`, `conda build` gives me this exception:

```
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml
```

So unfortunately we have to include all packages in `build` that are required during the `python setup.py install` process. :(